examples/pubsub/pub/pub.go

```diff
@@ -16,15 +16,16 @@ package main
 import (
 	"context"
 	"fmt"
-	"os"
 
 	dapr "github.com/dapr/go-sdk/client"
 )
 
 var (
 	// set the environment as instructions.
-	pubsubName = os.Getenv("DAPR_PUBSUB_NAME")
-	topicName  = "neworder"
+	pubsubName     = "messages"
+	bulkpubsubName = "bulkmessages"
```

Comment:
Declared but unused, did you duplicate this intending to use it in your example below?

Reply:
ya I should remove pubsubName = os.Getenv("DAPR_PUBSUB_NAME")

Reply:
Ya forgot to remove that , thanks for catching